### PR TITLE
Request to merge Feature/share password into main

### PR DESCRIPTION
**Pull request by Yczar**
* Added the share password functionality into the application

Screenshot_20210508-184755
Screenshot_20210508-184800

### CODE REVIEW

File: lib/components/bottom_sheet.dart [hunk 1]
```diff
@@ -27,7 +28,17 @@ class BottomSheetLayout extends GetView<HomeController> {
...
         Navigator.of(context).pop();
         break;
+      case "Share":
+        Share.share("""
+My ${_password.title} Details
 
+User Name: ${_password.username} 
+Password ${_password.password}
+
+Brought to you by Crypto Keep
```
Comment (singhsaubhik) [inline lib/components/bottom_sheet.dart]:
```suggestion
Sent from CryptoKeep
```

File: lib/components/bottom_sheet.dart [hunk 2]
```diff
@@ -27,7 +28,17 @@ class BottomSheetLayout extends GetView<HomeController> {
         AppSnackBar.show(context, text: PASSWORD_COPIED);
         Navigator.of(context).pop();
         break;
+      case "Share":
+        Share.share("""
+My ${_password.title} Details
 
+User Name: ${_password.username} 
+Password ${_password.password}
```
Comment (singhsaubhik) [inline lib/components/bottom_sheet.dart]:
```suggestion
Password: ${_password.password}
```

**Review by singhsaubhik — CHANGES_REQUESTED** [review]
Thanks for the contribution, I really appreciate. Apart from minor changes it looks good to me.

**Comment by Yczar** [comment]
I resolved them already
Glad it looks good to you.. hopefully when I think of any feature, I can always create a pull request

**Comment by Yczar** [comment]
I've made those changes

**Review by singhsaubhik — APPROVED** [review]

**Pull request merged**